release_build_files/readme.md

```diff
@@ -53,9 +53,8 @@ libs/ios/ARCH             | iOS static libraries (compiled against
 libs/android/ARCH/STL     | Android (GCC 4.8+ compatible) static
 |                         | libraries for each architecture and STL
 |                         | variant.
-|                         | _STL variants available:_
+|                         | _STL variant available:_
 |                         | * `c++`: LLVM libc++ runtime (recommended)
```

Comment:
Drive-by comment: Remove "(recommended)" from this line, since it's the _only_ option now.